src/test/java/com/amazon/ion/Ion_1_1_RoundTripTest.kt

```diff
@@ -243,194 +244,176 @@ class Ion_1_1_RoundTripTest {
         override val writerFn: (OutputStream) -> IonWriter = WRITER_INTERNED_DELIMITED
         override val readerFn: (ByteArray) -> IonReader = READER_NON_CONTINUABLE_STREAM_16
     }
-}
-
-/**
- * Base class that contains text-specific cases
- */
-abstract class Ion_1_1_RoundTripTextBase : Ion_1_1_RoundTripBase() {
-    abstract val newWriterForAppendable: (Appendable) -> IonWriter
-    override val readerFn: (ByteArray) -> IonReader = IonReaderBuilder.standard()::build
-
-    @ParameterizedTest(name = "{0}")
-    @MethodSource("testData")
-    fun testUserValuesSurviveRoundTripWrittenToAppendable(name: String, ion: ByteArray) {
-        val data: List<IonValue> = ION.loader.load(ion)
-        val appendable = StringBuilder()
-        val writer = newWriterForAppendable(appendable)
-        data.forEach { it.writeTo(writer) }
-        writer.close()
-        val actual = appendable.toString()
-
-        if (DEBUG_MODE) {
-            println("Expected:")
-            ion.printDisplayString()
-            println("Actual:")
-            println(actual)
-        }
-
-        assertReadersHaveEquivalentValues(
-            ION.newReader(ion),
-            ION.newReader(actual)
-        )
-    }
-}
 
-@OptIn(ExperimentalStdlibApi::class)
-abstract class Ion_1_1_RoundTripBase {
+    // Macro-aware Ion 1.1 transcode
 
-    abstract val writerFn: (OutputStream) -> IonWriter
-    abstract val readerFn: (ByteArray) -> IonReader
-    val systemReaderFn: (ByteArray) -> IonReader = ION::newSystemReader
+    @Nested
+    inner class BinaryMacroAwareTranscode_ReaderNonContinuableBufferDefault {
```

Comment:
This is the only thing I added in here, in order to test that a macro-aware transcode off all the test data results in data-model-equivalent results.